maestro-app/src/main/java/bio/overture/maestro/app/infra/adapter/outbound/metadata/study/song/SongStudyDAO.java

```diff
@@ -154,6 +155,16 @@ public SongStudyDAO(@NonNull WebClient webClient, @NonNull ApplicationProperties
             });
     }
 
+    private WebClient configWebClient() {
```

Comment:
it's not good to modify beans in classes, all configurations should be done in the configuration classes